### DISCUSSION TITLE
Use with Accessors?

### DISCUSSION THREAD

**Discussion by titantwentyone** [filamentphp/filament#540] [Help] [answered]
I have a calculated field on my Model `places_available`:

```
    public function getPlacesAvailableAttribute($value)
    {
        return 50 - $this->bookedPlaces()->count(); //bookedPlaces is a relationship
    }
```

I want to display this on the edit screen if possible. I tried the following in my resource:

```
    public static function form(Form $form)
    {
        return $form
            ->schema([
                Components\TextInput::make('places_available')
                    ->disabled()
            ]);
    }
```
But the field is empty.

I looked at the Placeholder component also but coiuldn't see a way to access the current record - only an arbitrary piece of data.

Any ideas on how to achieve this? Thank you in advance!

**Comment by danharrin** [comment, marked as answer]
You could try a `View` field, and then render `$formComponent->getLivewire()->record->accessor` in Blade or something? You'll be able to do this with a placeholder component in v2.

**Reply by titantwentyone** [reply]
Thanks @danharrin View field worked a treat!
